Documentation/network/servicemesh/envoy-load-balancing.rst

```diff
@@ -0,0 +1,116 @@
+
+Start a second terminal, then enable hubble port forwarding and observe
+traffic for the service ``echo-service``:
+
+.. code-block:: shell-session
+
+    $ kubectl -n kube-system port-forward deployment/hubble-relay 4245:4245 &
+    $ hubble observe --service echo-service -f
+
+
+You should be able to get a response from both of the backend services
+individually from ``client2``:
+
+.. code-block:: shell-session
+
+    $ kubectl exec -it $CLIENT -- curl -v echo-service:8080/
+
+Notice that Hubble shows all the flows between client pod and the backend pods
+via ``echo-service`` service.
+
+::
+
+    Jan 16 04:28:10.690: default/client-7dccb64ff6-t5gc7 (ID:5152) <> default/echo-service:8080 (world) pre-xlate-fwd TRACED (TCP)
+    Jan 16 04:28:10.690: default/echo-service:8080 (world) <> default/client-7dccb64ff6-t5gc7 (ID:5152) post-xlate-rev TRANSLATED (TCP)
+
+Add Proxy Load Balancing annotations to the services
+====================================================
+
+Adding a Layer 7 policy introduces the Envoy proxy into the path for this traffic.
+
+.. parsed-literal::
+
+    $ kubectl annotate service echo-service io.cilium.service/lb-l7=enabled
+    service/echo-service annotated
+
+Make a request to a backend service and observe the traffic with Hubble again:
+
+.. code-block:: shell-session
+
+    $ kubectl exec -it $CLIENT -- curl -v echo-service:8080/
+
+::
+
+    Jan 16 04:32:27.737: default/client-7dccb64ff6-t5gc7:56462 (ID:5152) -> default/echo-service:8080 (world) to-proxy FORWARDED (TCP Flags: SYN)
+    Jan 16 04:32:27.737: default/client-7dccb64ff6-t5gc7:56462 (ID:5152) <- default/echo-service:8080 (world) to-endpoint FORWARDED (TCP Flags: SYN, ACK)
+    Jan 16 04:32:27.737: default/client-7dccb64ff6-t5gc7:56462 (ID:5152) -> default/echo-service:8080 (world) to-proxy FORWARDED (TCP Flags: ACK)
+    Jan 16 04:32:27.737: default/client-7dccb64ff6-t5gc7:56462 (ID:5152) -> default/echo-service:8080 (world) to-proxy FORWARDED (TCP Flags: ACK, PSH)
+    Jan 16 04:32:27.739: default/client-7dccb64ff6-t5gc7:56462 (ID:5152) <- default/echo-service:8080 (world) to-endpoint FORWARDED (TCP Flags: ACK, PSH)
+    Jan 16 04:32:27.740: default/client-7dccb64ff6-t5gc7:56462 (ID:5152) -> default/echo-service:8080 (world) to-proxy FORWARDED (TCP Flags: ACK, FIN)
+    Jan 16 04:32:27.740: default/client-7dccb64ff6-t5gc7:56462 (ID:5152) <- default/echo-service:8080 (world) to-endpoint FORWARDED (TCP Flags: ACK, FIN)
+    Jan 16 04:32:27.740: default/client-7dccb64ff6-t5gc7:56462 (ID:5152) -> default/echo-service:8080 (world) to-proxy FORWARDED (TCP Flags: ACK)
+
+The request is now proxied through the Envoy proxy and then flows to the backend.
+
+Supported Annotations
+=====================
+
+.. list-table::
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Name
+     - Description
+     - Default Value
+   * - ``io.cilium.service/lb-l7``
+     - Enable L7 Load balancing for kubernetes service. Applicable values: ``envoy``, ``disabled``
```

Comment:
Nit: You could split this in two columns, description / applicable values. I don't mind much though.

Reply:
good point 💯